pkg/remote/aws/dynamodb_table_enumerator.go

```diff
@@ -0,0 +1,46 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+type DynamoDBTableEnumerator struct {
+	repository repository.DynamoDBRepository
+	factory    resource.ResourceFactory
+}
+
+func NewDynamoDBTableEnumerator(repository repository.DynamoDBRepository, factory resource.ResourceFactory) *DynamoDBTableEnumerator {
+	return &DynamoDBTableEnumerator{
+		repository,
+		factory,
+	}
+}
+
+func (e *DynamoDBTableEnumerator) SupportedType() resource.ResourceType {
+	return aws.AwsDynamodbTableResourceType
+}
+
+func (e *DynamoDBTableEnumerator) Enumerate() ([]resource.Resource, error) {
+	tables, err := e.repository.ListAllTables()
+	if err != nil {
+		return nil, remoteerror.NewResourceEnumerationError(err, string(e.SupportedType()))
+	}
+
+	results := make([]resource.Resource, len(tables))
+
+	for _, table := range tables {
+		results = append(
+			results,
+			e.factory.CreateAbstractResource(
+				string(e.SupportedType()),
+				*table,
```

Comment:
shouldn't we use aws.StringValue() ?

Reply:
I honestly don't know, according to other enumerators I think we do. Howether that means we could create a resource with empty string as ID 🤔

Reply:
Yes, also if `table` is `nil`, it'll not crash but use `""` as resource id, which sounds weird